src/components/filter.js

```diff
@@ -78,21 +79,24 @@ const Filter = ({
             )
         }
       })}
-      <ListItem
-        innerDivStyle={style.listItemInner}
-        leftIcon={<SettingsIcon style={style.listIcon} />}
-        primaryText={<FormattedMessage {...messages.changeFilters} />}
+      <FlatButton
```

Comment:
I am too. The `ListItem` wasn't firing events, presumably because it didn't have children, so I switched it to a button. I'll see what I can do to match the styling.